src/components/Startpage/Startpage.tsx

```diff
@@ -230,8 +245,8 @@ function Startpage() {
                     </Card.Header>
                     <Card.Body>
                         <p>
-                            We moved auctions to another server. 
-                            This should increase performance. 
+                            We moved auctions to another server.
+                            This should increase performance.
```

Comment:
why has this changed? It is out of date. 
May be changed to `hypixel api is offline`